docs/source/links/state.md

```diff
@@ -0,0 +1,295 @@
+---
+title: apollo-link-state
+description: Manage your local data with Apollo Client
+---
+
+Managing remote data from an external API is simple with Apollo Client, but
+where do we put all of our data that doesn't fit in that category? Nearly all
+apps need to centralize client-side data from user interactions and device APIs
+somewhere!
+
+In the past, Apollo users stored their application's local data in a separate
+Redux or MobX store. With `apollo-link-state`, you no longer have to maintain a
+second store for local state. Your Apollo Client cache is your single source of
+truth that holds all of your local data alongside your remote data. To access or
+update your local state, you use GraphQL queries & mutations just like you would
+for data from a server.
+
+When you use Apollo Client to manage your local state, you get all of the same
+benefits you know and love like caching, optimistic UI, and persistence without
```

Comment:
I don't think optimistic UI makes sense as a top-level feature here.

Also, I think we should watch out for the word "persistence" being overloaded for persisted queries and storing data persistently... not sure how to solve that one.

Reply:
Which features would you call out instead?

I'm trying to highlight things that are difficult w/ Redux but easy w/ Apollo - optimistic UI definitely fits that category. Do you think `offline persistence` sounds less ambiguous?

Reply:
> Do you think offline persistence sounds less ambiguous?

Yep!

> I'm trying to highlight things that are difficult w/ Redux but easy w/ Apollo

Yeah for sure, I just don't think optimistic UI for client side data is that interesting. Maybe we can talk about being able to easily attach local state to data you got from the server instead? I think the biggest win is client+server data in one place. OH! also, you can use GraphQL instead of writing janky selectors and it's efficient by default.